### PR TITLE
S2 product listing

### PR DESCRIPTION
**Pull request by HarshithaKonduru214**
Hi everyone, pls review my product listing page and let me know if anything can be improved.

Deploy link: <a>https://confident-bose-22822d.netlify.app/pages/product-listing/product-listing.html</a>

Thanks

### CODE REVIEW

**Review by anupkgurung — APPROVED** [review]
looks good.

File: pages/product-listing/product-listing.css [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+.card-badge {
+    position: absolute;
+    right: 0;
+    top: 0.5rem;
+    padding: 0.5rem;
+    color: var(--gray-text);
+    font-weight: bold;
+    background-color:#e0e7ff;
+}
```
Comment (rohanmathur91) [inline pages/product-listing/product-listing.css]: Have a variable for this color.

**Review by rohanmathur91 — APPROVED** [review]
Looks good, really liked the navbar.

**Comment by Wsn-195** [comment]
Looking awesome, Perfectly aligned as per P0 requirement.

File: pages/product-listing/product-listing.css (hunk 1 above)
Comment (HarshithaKonduru214) [reply pages/product-listing/product-listing.css]: I have updated, thanks for pointing it out :)

**Pull request merged**